node_modules/gh-orgunit/lib/rest.js

```diff
@@ -578,3 +581,43 @@ var exportOrgUnit = function(req, res) {
 
 GrassHopper.globalAdminRouter.on('get', '/api/orgunit/:id/export', exportOrgUnit);
 GrassHopper.appRouter.on('get', '/api/orgunit/:id/export', exportOrgUnit);
+
+/**
+ * @REST importOrgUnit
+ *
+ * Import a set of given organisational units, event series and events into an organisational unit
+ *
+ * @Server      admin,app
+ * @Method      POST
+ * @Path        /orgunit/{id}/import
+ * @PathParam   {number}            id              The id of the organisational unit into which to import the data
+ * @FormParam   {string}            data            The organisational units, event series and events to import
+ * @FormParam   {boolean}           deleteMissing   Whether to delete items from the database that aren't in the import data
+ */
+var importOrgUnit = function(req, res) {
+    OrgUnitImport.importOrgUnit(req.ctx, req.params.id, req.body.data, req.body.deleteMissing, function(errors) {
+        if (errors) {
+            // Because of the large potential of validation errors, the import API returns everything
+            // that is wrong with the provided data. We provide all the information to the user
+            if (_.isArray(errors)) {
+                var body = _.map(errors, function(error) {
+                    return error.msg;
+                }).join('\n');
+                var code = 400;
+                if (errors.length === 1) {
+                    code = errors[0].code;
+                }
+                return res.status(code).send(body);
```

Comment:
I think for the whole `errors.length === 1` case, we could just take `errors[0]` and do what happens in the `else` case.